### PR TITLE
migrate test_cassandra_mod to pytest

### PR DESCRIPTION
**Pull request by frogunder**
### What does this PR do?

migrate test_cassandra_mod to pytest

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [x] Tests written/updated

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: tests/pytests/unit/modules/test_cassandra_mod.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    Test for Return existing keyspaces
+    """
+    mock_keyspaces = ["A", "B", "C", "D"]
+
+    class MockSystemManager:
+        def list_keyspaces(self):
+            return mock_keyspaces
+
+    @pytest.fixture
+    def configure_keyspaces():
```
Comment (s0undt3ch) [inline tests/pytests/unit/modules/test_cassandra_mod.py:83]: This fixture is not really being used, right?

File: tests/pytests/unit/modules/test_cassandra_mod.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        def list_keyspaces(self):
+            return mock_keyspaces
+
+        def get_keyspace_column_families(self, keyspace):
+            if keyspace == "A":
+                return {"a": "saltines", "b": "biscuits"}
+            if keyspace == "B":
+                return {"c": "cheese", "d": "crackers"}
+
+    @pytest.fixture
```
Comment (s0undt3ch) [inline tests/pytests/unit/modules/test_cassandra_mod.py:108]: Same for this one...

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/pytests/unit/modules/test_cassandra_mod.py (hunk 1 above)
Comment (frogunder) [reply tests/pytests/unit/modules/test_cassandra_mod.py]: Removed, thanks.

File: tests/pytests/unit/modules/test_cassandra_mod.py (hunk 2 above)
Comment (frogunder) [reply tests/pytests/unit/modules/test_cassandra_mod.py]: Removed, thanks.

**Review by s0undt3ch — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Pull request merged**